### PR TITLE
Add welcome banner

### PR DESCRIPTION
**Pull request by jtrussell**
Adding this welcome banner thing. I moved the logout link to the profile page to be as close to the designs as possible.

### CODE REVIEW

**Comment by jtrussell** [comment]
Screen Shot 2019-09-06 at 10 49 23 PM

Screen Shot 2019-09-06 at 10 50 46 PM

File: next/components/welcome-banner.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          content: '\u25bc';
+          margin-left: 3px;
+          position: relative;
+          top: 1px;
+        }
+
+        .welcome-banner-link {
+          text-decoration: none;
+          font-weight: 600;
+          color: ${theme.background};
```
Comment (alexmiddeleer) [inline next/components/welcome-banner.js]: I suggest making a new variable name for this. Maybe `theme.invertedFontColor` or something.

**Comment by jtrussell** [comment]
Dinna fash, @lily-pytel - at this point I just assume anyone who wants to review will - I haven't been added specific reviewers recently. Happy to get this in before conflicts ;).

File: next/components/welcome-banner.js (hunk 1 above)
Comment (jtrussell) [reply next/components/welcome-banner.js]: Happy to do so - my thinking though was that making it the same color as the background was a feature. Intentional contrast if you will.

Comment (jtrussell) [reply next/components/welcome-banner.js]: I'll shift things around post-merge so I can unblock @lily-pytel.

**Pull request merged**